### PR TITLE
MAP-1941 connect redis upgrade to 8 and esLint upgrade from 7 to 8 lint

### PR DESCRIPTION
**Pull request by marcusaleman**
It is needed  to update Template Typescript code to align to the last dependencies update and modernise code with Redis 8.

Reference Template Typescript https://github.com/ministryofjustice/hmpps-template-typescript/pull/492

Code has been tested locally.

### CODE REVIEW

File: jobs/eventsPusher.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default class EventsPusher {
   getCredentials = (serviceAccountKey: string): unknown => {
     try {
       return JSON.parse(serviceAccountKey)
+      // eslint-disable-next-line  @typescript-eslint/no-unused-vars
```
Comment (marcusaleman) [inline jobs/eventsPusher.ts:20]: we need to disable some lint checkings in order to not change code lines

File: jobs/eventsPusher.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export default class EventsPusher {
...
     } catch (e) {
```
Comment (Thource) [inline jobs/eventsPusher.ts:21]:
```suggestion
    } catch (_e) {
```

File: server/services/healthCheck.ts [hunk 3]
```diff
@@ -37,8 +37,9 @@ function addAppInfo(result: HealthCheckResult): HealthCheckResult {
 
 function getBuild() {
   try {
-    // eslint-disable-next-line import/no-unresolved,global-require
+    // eslint-disable-next-line import/no-unresolved,global-require, @typescript-eslint/no-require-imports
     return require('../../build-info.json')
+    // eslint-disable-next-line  @typescript-eslint/no-unused-vars
   } catch (ex) {
```
Comment (Thource) [inline server/services/healthCheck.ts:43]:
```suggestion
  } catch (_ex) {
```

File: server/services/healthCheck.ts [hunk 4]
```diff
@@ -37,8 +37,9 @@ function addAppInfo(result: HealthCheckResult): HealthCheckResult {
 
 function getBuild() {
   try {
-    // eslint-disable-next-line import/no-unresolved,global-require
+    // eslint-disable-next-line import/no-unresolved,global-require, @typescript-eslint/no-require-imports
     return require('../../build-info.json')
```
Comment (Thource) [inline server/services/healthCheck.ts:41]: Could we change this to an import, rather than just ignoring eslint?

File: jobs/eventsPusher.ts (hunk 2 above)
Comment (marcusaleman) [reply jobs/eventsPusher.ts:21]: $npm run lint -> /GITHUB/WPIP/hmpps-welcome-people-into-prison-ui/jobs/eventsPusher.ts
  20:14  warning  '_e' is defined but never used  @typescript-eslint/no-unused-vars

File: server/services/healthCheck.ts [hunk 5]
```diff
@@ -37,9 +37,10 @@ function addAppInfo(result: HealthCheckResult): HealthCheckResult {
...
-  } catch (ex) {
+    // eslint-disable-next-line  @typescript-eslint/no-unused-vars
```
Comment (marcusaleman) [inline server/services/healthCheck.ts:42]: it is required to leave //eslint-dsiable , i refactor ex to _ex to follow patterns for this non used vars

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**